### ISSUE TITLE
Directory or repository of third-party scripts and such

### ISSUE THREAD

**Issue by eater** [i3/i3#3612]
## I'm submitting a…

<pre>
[ ] Bug
[ ] Feature Request
[x] Documentation Request
[ ] Other (Please describe in detail)
</pre>

The design philosophy of i3 intentionally sets out that all non-foundational functionality should be implemented via IPC or other tools external to the codebase. That's great! 

In order to encourage and streamline that, and make the end-user experience better, it would be incredibly helpful if there were a single place where users could find scripts that have already been created, and share new ones that they have implemented. The current system involves searching Reddit and maybe asking on IRC, and neither of those methods provides very good coverage of what actually exists.

Many other projects have simple directories of add-ons. Random example: https://github.com/todotxt/todo.txt-cli/wiki/Todo.sh-Add-on-Directory.  It could be hosted as a Github wiki page, and I volunteer to help maintain that.

**Comment by Airblader** [comment]
We do have the user-contributed articles section on the website, but maintaining that is pretty painful. I'm fairly sure we agreed in the past that keeping community-maintained scripts somewhere is a great idea that we support, though I'm not sure right now where we had that discussion.

Ultimately, of course, nothing stops anyone from creating such a repository. I don't think we'd want to host a repository in the i3 organization ourselves because it'd be a lot of effort to maintain it. I don't see an issue with linking to such a repository from the website and/or reddit, though. @stapelberg Your thoughts?

**Comment by stapelberg** [comment]
As long as it’s clearly marked that the directory/list is community-maintained, I’d be happy to link to it.

I do agree that it would be preferable for such a directory/list to live outside the official i3 org for the time being. If it becomes a no-brainer at some point to bless it as official, I’m happy to do that, but I want to be cautious with regards to what we sign up to maintain :).

Does that make sense?

**Comment by Airblader** [comment]
Great. Let's keep this open and if someone wants to start such a repository they can let us know so we can link to it in appropriate places.

**Comment by eater** [comment]
All right, I have kicked this off here: https://github.com/noodlor/i3-resources

I look forward to feedback!

**Comment by Airblader** [comment]
@eater Thanks! Feel free to send a PR to document this in places that seem useful. You can also post about it on our subreddit, we can make it sticky there.

**Comment by orestisfl** [comment]
We can mention https://github.com/altdesktop/i3ipc-python/tree/master/examples somewhere

**Comment by ghost** [comment]
@Airblader, "Great. Let's keep this open and if someone wants to start such a repository they can let us know so we can link to it in appropriate places."

Alright, keep this issue open please. I'll see what I can do about this when I have some more time. 
Most likley it will be done on GitHub somewhere so everyone can contribute to it.

**Comment by Airblader** [comment]
Thanks! Please note that the intention should be to maintain such a list long-term for us to link to it long-term.

**Comment by ghost** [comment]
@Airblader, of course. I have an interest in this as I'm now a new i3 user and love it.
I don't see me using any other WM now. It supports all my four monitors on two cards well.
I'm still learning everything in i3 but I have some good experience in programming websites.
